src/ActionList/Item.tsx

```diff
@@ -125,22 +166,50 @@ const DescriptionContainer = styled.span`
 /**
  * An actionable or selectable `Item` with an optional icon and description.
  */
-export function Item({
-  text,
-  description,
-  descriptionVariant = 'inline',
-  selected,
-  leadingVisual: LeadingVisual,
-  trailingIcon: TrailingIcon,
-  trailingText,
-  variant = 'default',
-  ...props
-}: Partial<ItemProps> & {item?: ItemInput}): JSX.Element {
+export function Item(itemProps: Partial<ItemProps> & {item?: ItemInput}): JSX.Element {
+  const {
+    text,
+    description,
+    descriptionVariant = 'inline',
+    selected,
+    leadingVisual: LeadingVisual,
+    trailingIcon: TrailingIcon,
+    trailingText,
+    variant = 'default',
+    disabled,
+    onAction,
+    onKeyPress,
+    onClick,
+    ...props
+  } = itemProps
   return (
-    <StyledItem tabIndex={-1} variant={variant} aria-selected={selected} {...props}>
+    <StyledItem
+      tabIndex={disabled ? undefined : -1}
+      variant={variant}
+      aria-selected={selected}
+      {...props}
+      onKeyPress={event => {
```

Comment:
Should this anonymous function (and the `onClick` value below) become replaced with a `React.useCallback`?

Reply:
done!

Reply:
Fixed in https://github.com/primer/components/pull/1186/commits/052c8dddc1b21c9e2fb94e5be06b7f0c27ffd0b4